app/assets/javascripts/components/all_playlists.js.jsx

```diff
@@ -2,7 +2,7 @@ class AllPlaylists extends React.Component {
 
   render () {
     var playlists = this.props.playlists.map( (playlist) => {
-      return <Playlist key= {playlist.table.id} playlist={playlist} />
+      return <Playlist key= {playlist.id} playlist={playlist} />
```

Comment:
I find this sort of thing to be more readable if you just place the mapping inside the JSX where it goes instead of the variable, since you don't have to scroll back up to a variable, or make a variable for that matter. up to you